test/greet-test.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const greeting = require('../lib/greet.js');
+const expect = require('chai').expect;
+
+describe('Greeting', function() {
+  describe('#greet', function() {
+    it('should return hello britt', function() {
+      let result = greeting.greet('britt');
+      expect(greeting).to.have.property('greet');
```

Comment:
Nice to see you being mindful of testing for a property 'greet' from your greeting module!